packages/hardhat-etherscan/src/network/prober.ts

```diff
@@ -40,6 +40,9 @@ enum NetworkID {
   // Arbitrum
   ARBITRUM_ONE = 42161,
   ARBITRUM_TESTNET = 421611,
+  // Avalanche
+  AVALANCHE = 43114,
+  AVALANCHE_FUJI_TESNET = 43113,
```

Comment:
Was just scrolling through this PR. There is a typo ("Tesnet") instead of Testnet for both changes!

Reply:
Thaks, just fixed.